test/src/test/java/hudson/scm/ChangeLogSetTest.java

```diff
@@ -30,7 +31,7 @@ public void catchingExceptionDuringAnnotation() {
             notCaught = true;
         }
         assertEquals((new EntryImpl()).getMsg(), change.getMsg());
-        assertEquals(false, notCaught);
+        assertFalse(notCaught);
```

Comment:
Maybe kill the `notCaught` variable and use `fail(t.getMessage())` in the catch block instead.

Reply:
You are right - that is much clearer.